src/frontend/data/sideEffects/upload/index.ts

```diff
@@ -66,10 +67,25 @@ export const upload = (
       notifyObjectUploadProgress,
     );
 
-    await addResource(objectType, {
-      ...object,
-      upload_state: uploadState.PROCESSING,
-    });
+    if (object.hasOwnProperty('title')) {
+      await addResource(objectType, {
```

Comment:
This is unrelated to this PR itself but I just noticed the order of the arguments is reversed between `addResource` and `updateResource`. I know Typescript smoothes this out but maybe we can make an issue to have the same order in all these methods?

Reply:
This is because the php api inconsistency is missing me

Reply:
😱